### PR TITLE
add tests opencensus

### LINKED ISSUE #604: Tests for opencensus module
There are no tests for this module now, it would be good to have some to be sure that nothing breaks when making code changes.

### PR DESCRIPTION
**Pull request by IvanFinochenko**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -57,9 +57,10 @@ object Dependencies {
   )
 
   lazy val opencensus = zio ++ Seq(
-    Orgs.opencensus % "opencensus-api"               % Versions.opencensus,
-    Orgs.opencensus % "opencensus-impl"              % Versions.opencensus,
-    Orgs.opencensus % "opencensus-contrib-http-util" % Versions.opencensus
+    Orgs.opencensus        % "opencensus-api"               % Versions.opencensus,
+    Orgs.opencensus        % "opencensus-impl"              % Versions.opencensus,
+    Orgs.opencensus        % "opencensus-contrib-http-util" % Versions.opencensus,
+    Orgs.scalaLangModules %% "scala-collection-compat"      % Versions.scalaCollectionCompat
```
Comment (grouzen) [inline project/Dependencies.scala]: I assume we need this dep for tests only, so in this case it would be nice to add `% Test`.

**Comment by grouzen** [comment]
Thanks for your contribution @IvanFinochenko!

File: opencensus/src/test/scala/zio/telemetry/opencensus/OpenCensusTracingTest.scala [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package zio.telemetry.opencensus
+
+import io.opencensus.trace.`export`.SpanData
+import io.opencensus.trace.samplers.Samplers
+import io.opencensus.trace.{ SpanId, Tracer, Tracing => OTracing }
+import zio._
+import zio.test.Assertion._
+import zio.test._
+
+object OpenCensusTracingTest extends ZIOSpecDefault {
```
Comment (grouzen) [inline opencensus/src/test/scala/zio/telemetry/opencensus/OpenCensusTracingTest.scala]: Could you please rename it to `TracingTest` in order to keep the same naming style across all modules?

File: project/Dependencies.scala (hunk 1 above)
Comment (IvanFinochenko) [reply project/Dependencies.scala]: done

File: opencensus/src/test/scala/zio/telemetry/opencensus/OpenCensusTracingTest.scala (hunk 2 above)
Comment (IvanFinochenko) [reply opencensus/src/test/scala/zio/telemetry/opencensus/OpenCensusTracingTest.scala]: done

**Review by grouzen — DISMISSED** [review]

**Comment by grouzen** [comment]
> [CLA assistant check](https://cla-assistant.io/zio/zio-telemetry?pullRequest=682) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/zio/zio-telemetry?pullRequest=682) before we can accept your contribution.
> **i.a.finochenko** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/zio/zio-telemetry?pullRequest=682) it.

@IvanFinochenko There left the last step where you need to sign the license agreement.

**Comment by grouzen** [comment]
@IvanFinochenko I think to make CLA checker happy, you need to rebase this branch so that all commits have the same author.

**Pull request closed without merging**